lib/views/list-item/index.js

```diff
@@ -207,9 +227,7 @@ class ListItem extends React.Component {
     debug('gesture end', vx, exceededThreshold);
 
     // fast swipes fling the tile away
-    if (exceededThreshold) {
-      this.swipeAway(vx);
-    }
+    if (exceededThreshold) this.swipeAway(vx);
```

Comment:
That's why i'm not blocking this pr ... i was just expressing my angriness ... you own me a ☕